Resources/Locale/en-US/devices/device-network.ftl

```diff
@@ -10,7 +10,7 @@ device-frequency-prototype-name-basic-device = Basic Devices
 device-frequency-prototype-name-cyborg-control = Cyborg Control
 device-frequency-prototype-name-robotics-console = Robotics Console
 
-## camera frequencies
+# camera frequencies
```

Comment:
These *.ftl changes are irrelevant to the PR and should be removed.

Reply:
They are not irrelevant to the PR: "that would show up as errors with the fluent extension."